clib/feldspar_array.h

```diff
@@ -50,13 +50,10 @@ struct array
     int32_t  length;   /* number of elements in the array */
```

Comment:
Doesn't this change eliminate the need for struct array?

Reply:
Not in the "ivar_X" functions.